### ISSUE TITLE
Like and dislike buttons + another proposals

### ISSUE THREAD

**Issue by allaklimova** [graphprotocol/mission-control-curator#311]
In order to increase the engagement of participants, I suggest adding **like** and **dislike** buttons right below the project image. Also, we can add links to the project telegram channel, plus information about the presence of project subgraphs.

**Comment by yanivtal** [comment]
Yes this could be an interesting way to establish reputation for projects.
